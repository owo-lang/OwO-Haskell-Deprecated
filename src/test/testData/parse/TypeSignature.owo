module ParseTest.TypeSignature where

  a : b
  c : d
