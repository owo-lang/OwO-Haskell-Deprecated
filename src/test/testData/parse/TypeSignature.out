File type: CodeFileType
Top module name: .ParseTest.TypeSignature
 type signature a (40 3 3) (41 3 4)
  no pragmas
  named reference b (44 3 7) (45 3 8)
 type signature c (48 4 3) (49 4 4)
  no pragmas
  named reference d (52 4 7) (53 4 8)
